### PR TITLE
Redirect the HTTPS overview

### PR DESCRIPTION
**Pull request by kaycebasques**
What's changed, or what was fixed?

- See https://github.com/GoogleChrome/web.dev/issues/2530

**Fixes:** https://github.com/GoogleChrome/web.dev/issues/2530

**Target Live Date:** Q2 2020

- [x] This has been reviewed and approved by @kaycebasques
- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [ ] I've staged the site and manually verified that my content displays correctly.

### CODE REVIEW

File: src/content/en/fundamentals/security/_redirects.yaml [hunk 1]
```diff
@@ -14,3 +14,6 @@ redirects:
 
 - from: /web/fundamentals/security/credential-management/store-credentials
   to: /web/fundamentals/security/credential-management/save-forms#store_the_credential
+
+- from: /web/fundamentals/security/encrypt-in-transit/why-https
+  to: https://web.dev/why-https-matters
```
Comment (kaycebasques) [inline src/content/en/fundamentals/security/_redirects.yaml:19]: This isn't working on staging but I'm pretty sure it's set up correctly and will work in production

File: src/content/en/fundamentals/security/_toc.yaml [hunk 2]
```diff
@@ -8,8 +8,9 @@ toc:
       path: /web/fundamentals/security/csp/
     - title: Encrypting Data In Transit
       section:
-      - title: Why HTTPS
+      - title: Why HTTPS matters
         path: /web/fundamentals/security/encrypt-in-transit/why-https
```
Comment (petele) [inline src/content/en/fundamentals/security/_toc.yaml:12]: can you set the URL here to the web.dev URL so it doesn't have to go through the redirect?

Comment (petele) [inline src/content/en/fundamentals/security/_toc.yaml:12]:
```suggestion
        path: https://web.dev/why-https-matters
```

Set the path to the actual new path so it goes directly there and doesn't have to go through the redirect.

**Comment by kaycebasques** [comment]
Thanks @petele

**Pull request merged**